quiz.py

```diff
@@ -0,0 +1,148 @@
+import random
+from abc import ABCMeta, abstractmethod
+from sqlalchemy.sql.expression import exists
+from sqlalchemy.sql.functions import func
+from models.painter import Painter
+from models.painting import Painting
+from models.saint import Saint
+
+
+class MultipleChoiceQuestion:
+    def __init__(self, element, correct_option, option_list):
+        self._element = element
+        self.element = element.values()[0]
+        self.correct_option = correct_option
+        self.option_list = option_list
+
+
+class Quiz:
+    __metaclass__ = ABCMeta
+
+    def __init__(self):
+        self.number_of_rounds = 0
+        self.score = []
+
+        # current
+        self.current_question = None
+
+        self.right_guesses = sum(self.score)
+        self.wrong_guesses = len(self.score) - self.right_guesses
+
+    def get_score(self):
```

Comment:
this getter is not used at all in code. please remove. also, afaik, it's not pythonic to create getters, see: 
http://stackoverflow.com/questions/2627002/whats-the-pythonic-way-to-use-getters-and-setters
http://stackoverflow.com/questions/6618002/python-property-versus-getters-and-setters